fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeJobV2.java

```diff
@@ -1013,6 +1024,9 @@ private void cancelInternal() {
                         tbl.deleteIndexInfo(shadowIndexName);
                     }
                     tbl.setState(OlapTableState.NORMAL);
+                    if (disableReplicatedStorageForGIN) {
+                        tbl.setEnableReplicatedStorage(true);
```

Comment:
replicated storage may be not enabled before.

Reply:
> replicated storage may be not enabled before.

if replicated storage is not enable before, `disableReplicatedStoreageForGIN` is false.